src/lib/admin/QuestionDetailView.svelte

```
@@ -280,46 +363,82 @@
+					class="text-xs font-semibold uppercase tracking-wide text-base-content/60"
+					class:mb-2={isMobile}
+					class:mb-3={!isMobile}
+				>
+					Rationale
+				</div>
+				<div
+					class="rounded-2xl border border-base-300 bg-base-200/30"
+					class:p-3={isMobile}
+					class:p-4={!isMobile}
+				>
+					<div class="text-sm text-base-content/80 tiptap-content">{@html questionRationale}</div>
```

ESLint

[error] `{`@html`}` can lead to XSS attack.

(svelte/no-at-html-tags)